move-mutator/tests/move-assets/simple_move_2_features/Move.toml

```diff
@@ -3,7 +3,7 @@ name = "simple_move_2_features"
 version = "0.0.0"
 
 [dependencies]
-AptosStdlib = { git = "https://github.com/aptos-labs/aptos-framework.git", subdir = "aptos-stdlib", rev = "main" }
+AptosStdlib = { git = "https://github.com/aptos-labs/aptos-framework.git", subdir = "aptos-stdlib", rev = "aptos-release-v1.35" }
```

Comment:
I had to do this, because of [these](https://eig3r.slack.com/archives/C060M9W35LG/p1758869727526159?thread_ts=1758869521.320289&cid=C060M9W35LG) Move errors. 
To me they seem like they're a but on Aptos's side. Correct me if I'm wrong here.